Formula/rancher-compose.rb

```diff
@@ -1,26 +1,14 @@
 class RancherCompose < Formula
-  desc "Docker Compose compatible client to deploy to Rancher"
-  homepage "https://github.com/rancher/rancher-compose"
-  url "https://github.com/rancher/rancher-compose/archive/v0.7.4.tar.gz"
-  sha256 "a0d285dd0608323c28a78782e85fbe3f52ddaad9d101030fad39efe3b20dadc9"
-
-  bottle do
```

Comment:
Please don't wipe bottle blocks across updates, our CI knows to ignore existing ones where the formula changes.